statsmodels/graphics/regressionplots.py

```diff
@@ -1274,7 +1274,8 @@ def added_variable_resids(results, focus_exog, resid_type=None,
     if fit_kwargs is not None:
         args.update(fit_kwargs)
     new_result = new_model.fit(**args)
-    if not new_result.converged:
+    cnvrg = new_result.converged if isinstance(model, (GEE, GLM)) else True
```

Comment:
any reason not to just `if not getattr(new_result, "converged", True)` so that future models would work without explicit whitelisting?

Reply:
Agreed, I made the change.